lib/browser/api/web-contents.js

```diff
@@ -104,6 +104,16 @@ WebContents.prototype.sendToAll = function (channel, ...args) {
   return this._send(true, channel, args)
 }
 
+WebContents.prototype.sendSync = function (channel, ...args) {
+  const requestId = getNextId()
+  return new Promise((resolve, reject) => {
```

Comment:
I'm confused here, if this is a promise how can it be synxronous.  Surely this is an async thing

Reply:
Maybe sendWithReply is a better name.